scopesim_templates/metis/__init__.py

```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
+
+from laser import laser_spectrum
+from pinhole_mask import pinhole_mask
```

Comment:
Wouldn't these also need the `@add_function_call_str` decorator? That is necessary to get the header keywords right. But I forgot exactly how this works and I didn't document the why...

Also, the reason these imports are in `__init__` is that this makes them shorter so they are easier to fit in a non-CONTINUE keyword.

Reply:
added it 👍